src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -47,5 +47,8 @@ public static class EnvironmentSettingNames
         public const string AzureWebsiteAltZipDeployment = "WEBSITE_RUN_FROM_ZIP";
         public const string AzureWebsiteRunFromPackage = "WEBSITE_RUN_FROM_PACKAGE";
         public const string RegionName = "REGION_NAME";
+
+        public const string AzureAppServiceStorage = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
```

Comment:
This applies to Linux only, can we update the environment variable to reflect that?